content/tsColinPrecip.Rmd

```diff
@@ -86,28 +84,22 @@ precipMap <- function(precipData, startDate, endDate){
   precip_breaks <- c(seq(0,80,by = 10), 200)
   
   precipData_cols <- precipData %>% 
-    group_by(state_fullname, county_mapname) %>% 
+    group_by(statename, statecounty) %>% 
     summarize(cumprecip = sum(precipVal)) %>% 
-    mutate(cols = cut(cumprecip, breaks = precip_breaks, labels = cols, right=FALSE)) %>% 
+    mutate(cols = cut(cumprecip, breaks = precip_breaks, labels = cols, right=FALSE)) %>%
     mutate(cols = as.character(cols))
   
   par(mar = c(0,0,3,0))
   
   # png('tsColin.png', width = 7, height = 5, res = 150, units = 'in')
-  m1 <- map('county', regions = precipData_cols$state_fullname, col = "lightgrey")
-  m2 <- map('state', regions = precipData_cols$state_fullname, 
+  m1 <- map('county', regions = precipData_cols$statename, col = "lightgrey")
+  m2 <- map('state', regions = precipData_cols$statename, 
             add = TRUE, lwd = 1.5, col = "darkgrey")
   
-  # some county names are mismatched, only plot the ones that maps library 
-  # knows about and then order them the same as the map
-  precipData_cols <- precipData_cols %>%
-    mutate(county_mapname = gsub(x = county_mapname, pattern = 'saint', replacement = 'st')) %>%
-    mutate(county_mapname = gsub(x = county_mapname, pattern = 'okaloosa',
-                                 replacement = 'okaloosa:main')) %>%
-    filter(county_mapname %in% m1$names)
-  precipData_cols <- precipData_cols[na.omit(match(m1$names, precipData_cols$county_mapname)),]
-  
-  m3 <- map('county', regions = precipData_cols$county_mapname, 
+  # some county names are mismatched, order them the same as the map
+  precipData_cols <- precipData_cols[na.omit(match(m1$names, precipData_cols$statecounty)),]
```

Comment:
For some reason, they are not in the same order but they have all the same counties. This orders them so that they are the same. If they aren't in the correct order (even when you just plot m3) it looks like this:

```r
![image](https://cloud.githubusercontent.com/assets/13220910/20576781/f121925a-b184-11e6-91ac-5f0cdcadaddc.png)

```